lib/DeonData.ts

```diff
@@ -21,10 +21,15 @@ export interface InstantiationOutput {
   tag: string;
 }
 
-export interface ExpressionInput {
+export interface CheckExpressionInput {
   csl: string;
 }
 
+export interface EvaluateExpressionInput {
+  csl: string;
+  values: Value[];
```

Comment:
Could we make values optional to make this a non-breaking change?

Reply:
Yes -- that way it also corresponds more closely to the expected input at the webservice's endpoint.